pybossa/cache/projects.py

```diff
@@ -252,7 +252,7 @@ def search_lock_status_sorting_result():
             # if not sort by lock_status or locked_tasks_in_project is empty/None,
             # sort by the column "order_by"
             order_by =  args.get('order_by')
-            sql_order_by = 'id ASC' if not (locked_tasks_in_project and order_by) else order_by
+            sql_order_by = order_by if not locked_tasks_in_project and order_by else 'id ASC'
```

Comment:
The sort will not function if `locked_tasks_in_project` is TRUE. Perhaps, this line should be changed to:

```
sql_order_by = order_by or 'id ASC'
```